src/components/PullRequestList/index.tsx

```diff
@@ -73,6 +73,16 @@ const PullRequestList: FC<PullRequestListProps> = ({ prType }) => {
     );
   });
 
+  const { CancelToken } = axios;
+  let cancel;
+
+  axios.get('https://api.realdevsquad.com/user', {
+    cancelToken: new CancelToken((c) => {
+      cancel = c;
```

Comment:
You cannot assign a function scoped variable inside a callback scope.
The reason being, the callback is offloaded to a worker thread and the main thread progresses.
So it is essentially a race condition, whether the worker finishes before the main thread arrives at the utilization of the assigned part.
In this case it is guaranteed to fail, because of such less time to reach the utilization.
Learning the hard way will cement this in your memory like me 😂

Reply:
Yes, it was an outdated code.
 All the necessary changes have been made now. 
Thanks for your suggestions!